src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ParameterTest/TvpTest.cs

```diff
@@ -1399,7 +1399,7 @@ private void WriteReader(SqlDataReader rdr)
                 {
                     for (int i = 0; i < colCount; i++)
                     {
-                        Console.Write("{0}  ", rdr.GetValue(i));
+                        Console.Write("{0}  ", DateTimeVariantTest.GetValueString(rdr.GetValue(i)));
```

Comment:
Do we need to write all this info to logs?

Reply:
I also lots of other places in this file, we're writing out, please remove those too.. Since the test was disabled, it never caught eyes.

Reply:
These are written to `SqlParameterTest.out` and this out file is later compared with `SqlParameterTest_ReleaseMode.bsl`. This is the design of this TvpTest. `VerifyColumnBoundaries` is required to run `TestMain()`.

Reply:
Ok, that's fine then.